### PR TITLE
Support for libxc 7

### PR DESCRIPTION
**Pull request by vanderhe**
On top of #104.

Bug fix, as we claim compatibility with libxc >=6.0.0 in the README.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -12,32 +12,34 @@ jobs:
...
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        mpi: [nompi, openmpi]
+        mpi: [nompi, mpich]
+        libxc: [6, 7]
         config: [Debug]
-        version: [13]
+        exclude:
```
Comment (bhourahine) [inline .github/workflows/build.yml:26]: Possibly needs a comment as to why.

**Review by bhourahine — APPROVED** [review]
See the comment, but otherwise good.

File: .github/workflows/build.yml (hunk 1 above)
Comment (vanderhe) [reply .github/workflows/build.yml:26]: It is just to reduce $\mathrm{CO_2}$ emissions, as the serial binary is already tested in combination with libxc 7.

Comment (bhourahine) [reply .github/workflows/build.yml:26]:
```suggestion
        # Serial binary is tested with libxc 7
        exclude:
```

Comment (vanderhe) [reply .github/workflows/build.yml:29]: Incorporated.

**Pull request merged**